examples/07-EMIT/interference_classification.py

```diff
@@ -115,4 +115,132 @@ def interference_classification(emitapp, use_filter = False, filter = []):
         all_colors.append(rx_colors)
         power_matrix.append(rx_powers)
     
+    return all_colors, power_matrix
+
+def protection_level_classification(emitapp, global_protection_level = True, global_levels = [], protection_levels = {}, use_filter = False, filter = []):
+    """
+    Classify interference type as according to inband/inband, 
+    out of band/in band, inband/out of band, and out of band/out of band.
```

Comment:
```suggestion
    Classify interference type according to band type.
    
    Options for band type are `inband/inband, out of band/in band,
    inband/out of band, and out of band/out of band.
```
The summary description should be one line, with additional information providing after a blank line.
If the options are literals, they should all be formatted like this: ``"inband/inband"``